<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mailing</class>
 <widget class="QDialog" name="mailing">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>774</width>
    <height>444</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="envoyer">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>370</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
font-family: &quot;Roboto&quot;, sans-serif;
    text-transform: uppercase;
    outline: 0;
    background: #2c3e50;
    color: #FFFFFF;
    width: 100%;
    border: 0;
    padding: 15px;
    font-size: 14px;
    -webkit-transition: all 0.3 ease;
    transition: all 0.3 ease;
    cursor: pointer;

}
QPushButton:hover
{

    background: #3498db;
}




</string>
   </property>
   <property name="text">
    <string>Envoyer</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineedit_msg">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>120</y>
     <width>411</width>
     <height>231</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit
{
	background-color: #2c3e50;
	color: #fcfcfc;
	border-style: solid;
	border-width: 2px;
	border-color: #1e4b6d;


}


QLineEdit::hover
{
	background-color: #2c3e50;
	color: #fcfcfc;
	border-style: solid;
	border-width: 2px;
	border-color: #01dcff;


}


QLineEdit::focus
{
	
	color: #fcfcfc;
	border-style: solid;
	border-width: 2px;
	border-color: #01dcff;


}
</string>
   </property>
   <property name="placeholderText">
    <string>message</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineedit_sujet">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>80</y>
     <width>411</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit
{
	background-color: #2c3e50;
	color: #fcfcfc;
	border-style: solid;
	border-width: 2px;
	border-color: #1e4b6d;


}


QLineEdit::hover
{
	background-color: #2c3e50;
	color: #fcfcfc;
	border-style: solid;
	border-width: 2px;
	border-color: #01dcff;


}


QLineEdit::focus
{
	
	color: #fcfcfc;
	border-style: solid;
	border-width: 2px;
	border-color: #01dcff;


}
</string>
   </property>
   <property name="placeholderText">
    <string>le sujet</string>
   </property>
  </widget>
  <widget class="QPushButton" name="envoyer_2">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>370</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
font-family: &quot;Roboto&quot;, sans-serif;
    text-transform: uppercase;
    outline: 0;
    background: #2c3e50;
    color: #FFFFFF;
    width: 100%;
    border: 0;
    padding: 15px;
    font-size: 14px;
    -webkit-transition: all 0.3 ease;
    transition: all 0.3 ease;
    cursor: pointer;

}
QPushButton:hover
{

    background: #3498db;
}




</string>
   </property>
   <property name="text">
    <string>cancel</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="comboBox_3">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>40</y>
     <width>411</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit
{
	background-color: #2c3e50;
	color: #fcfcfc;
	border-style: solid;
	border-width: 2px;
	border-color: #1e4b6d;


}


QLineEdit::hover
{
	background-color: #2c3e50;
	color: #fcfcfc;
	border-style: solid;
	border-width: 2px;
	border-color: #01dcff;


}


QLineEdit::focus
{
	
	color: #fcfcfc;
	border-style: solid;
	border-width: 2px;
	border-color: #01dcff;


}
</string>
   </property>
   <property name="placeholderText">
    <string>destinaire</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>-100</x>
     <y>0</y>
     <width>871</width>
     <height>451</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/img/policeman-background_48369-15139.jpg) 1 1 1 1 stretch stretch;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label_10</zorder>
  <zorder>envoyer</zorder>
  <zorder>lineedit_msg</zorder>
  <zorder>lineedit_sujet</zorder>
  <zorder>envoyer_2</zorder>
  <zorder>comboBox_3</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
